priv/repo/migrations/20200316103722_create_user_auth_tables.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Demo.Repo.Migrations.CreateUserAuthTables do
+  use Ecto.Migration
+
+  def change do
+    execute "CREATE EXTENSION IF NOT EXISTS citext", ""
+
+    create table(:users) do
+      add :email, :citext, null: false
+      add :encrypted_password, :string, null: false
+      add :confirmed_at, :naive_datetime
+      timestamps()
+    end
+
+    create unique_index(:users, [:email])
+
+    create table(:user_tokens) do
+      add :user_id, references(:users, on_delete: :delete_all), null: false
+      add :token, :binary, null: false
+      add :context, :string, null: false
+      add :sent_to, :string
+      add :inserted_at, :naive_datetime
+    end
+
+    create unique_index(:user_tokens, [:token])
```

Comment:
@aaronrenner I have just changed this line. :) It is an important change for security.

Reply:
Wait a sec, I will change it again. :D

Reply:
Done!